Remove import resulting in 'Error: Cannot find module'

Our restructuring of the jsx templates via #2356, may have caused our `browserify` compiler to generate the following traceback error:

```bash
+ file_extension=jsx
+ '[' jsx = jsx ']'
+ cd /usr/lib/node_modules
+ browserify /vagrant/src/jsx/select_session.jsx -t '[' babelify --presets es2015,react ']' -o /vagrant/src/js/select_session.js
Error: Cannot find module './import/input-type/data_upload_new.jsx' from '/vagrant/src/jsx'
    at /usr/lib/node_modules/browserify/node_modules/resolve/lib/async.js:55:21
    at load (/usr/lib/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/lib/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at /usr/lib/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:82:15)
+ echo select_session.js
+ read -r file
```

**Note:** the above error, most likely correspond to the [traceback](https://travis-ci.org/jeff1evesque/machine-learning/builds/109255927#L1053-L1086) error we have been receiving with our travis ci implementation.

--- a/src/jsx/select_session.jsx
+++ b/src/jsx/select_session.jsx
@@ -11,8 +11,6 @@ import ModelGenerate from './import/session-type/model_generate.jsx';
 import ModelPredict from './import/session-type/model_predict.jsx';
 import DataNew from './import/session-type/data_new.jsx';
 import DataAppend from './import/session-type/data_append.jsx';
-import DataUploadNew from './import/input-type/data_upload_new.jsx';
-import DataUploadAppend from './import/input-type/data_upload_append.jsx';
 import Submit from './import/general/submit.jsx';
 import ResultDisplay from './import/result/result_display.jsx';
 import Spinner from './import/general/spinner.jsx';

